configure.ac

```diff
@@ -94,7 +94,7 @@ case "${host_os}" in
         ;;
     cygwin*)
         AM_CONDITIONAL([ADD_RT], false)
-        AM_CONDITIONAL([T_WIN], true)
+        AM_CONDITIONAL([T_WIN], false)
```

Comment:
`T_WIN` is set to `false` by default, so this line could simply be removed (if `false` is the correct value).